bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -183,6 +183,7 @@ object.invalid.final.field.update = cannot update ''final'' field ''{0}'' in obj
 unsupported.comparison.operation = cannot compare ''{0}'' and ''{1}''
 unordered.types.in.comparison = ''{0}'' is unordered with respect to ''{1}''
 incompatible.type.assignment = ''{0}'' cannot be assigned to type ''{1}''
+incompatible.java.object.type.assignment = value creation with ''{0}'' not supported: expected a BValue
```

Comment:
''{0}'' is not from a valid java runtime class. It should be sub class of one of following classes.

java.lang.Number
java.lang.Float 
or
package io.runtime.api.values.